### PR TITLE
feat(staff): create staff member

### PR DESCRIPTION
**Pull request by erezsob**

### CODE REVIEW

File: src/v0/staff.ts [hunk 1]
```diff
@@ -34,10 +68,27 @@ export class Staff {
...
       }
     })
   }
+
+  create(staffMember: StaffMember): Promise<StaffResponse> {
+    return new Promise(async (resolve, reject) => {
+      const uri = `${this.options.base}${this.endpoint}/${this.options.user}`
+      try {
+        const response = await this.http.getClient().post(uri, staffMember)
+        console.log('asdf', response.data.results[0])
```
Comment (jmy-kellogg) [inline src/v0/staff.ts]: can this be removed?

Comment (erezsob) [reply src/v0/staff.ts]: Oops. Removed. 
Thanks

Comment (jmy-kellogg) [reply src/v0/staff.ts]: lol. I do that all the time.

**Pull request merged**